app/models/google_spread_sheet.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require("bundler")
+Bundler.require
+
+class GoogleSpreadSheet
+
+  # Get google sheet credentials from AwsParameterStore.
+  def google_sheet_client
+    google_credentials = ENV['NEWS_LETTER_GOOGLE_CREDENTIALS']
+    session = GoogleDrive::Session.from_service_account_key(StringIO.new(google_credentials))
+    @spreadsheet = session.spreadsheet_by_title("MLN newsletter signup requests - #{ENV['RAILS_ENV']}")
+    @worksheet = @spreadsheet.worksheets.first
+  rescue StandardError => e
+    LogWrapper.log('ERROR', {'message' => "Error occcured while calling the google sheets. #{e.message}",
+                             'method' => 'google_sheet_client'})
+    raise "We've encountered an error and were unable to confirm your email"
```

Comment:
Is the wording of this message something that came down as a business requirement?  I'm thinking maybe we want to let the user know what to do:  do we want them to wait, or refresh, or send a complaint email to someone?

Reply:
@daryanypl Still messages  Stephen is going to give. As of now i added some placeholder messages.